test/functional/tests/lazy_writes/test_lazy_writes_clean.py

```diff
@@ -231,86 +231,119 @@ def test_clean_remove_core_with_fs(cache_mode, fs):
         remove(mnt_point, True, True, True)
 
 
+@pytest.mark.parametrize("cache_mode", CacheMode.with_traits(CacheModeTrait.LazyWrites))
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_clean_remove_core_without_fs(cache_mode):
+    """
+        title: Test of the ability to remove core from cache in modes with lazy writes.
+        description: |
+          Test if OpenCAS removes core with in modes with lazy writes without data loss.
```

Comment:
add info to the title and description, that the test handles the case without fs.